src/axom/slic/docs/sphinx/sections/synopsis.rst

```diff
@@ -0,0 +1,59 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+Slic is a component of the `Axom Toolkit`_, currently in development at
+`Lawrence Livermore National Laboratory (LLNL) <http://www.llnl.gov>`_.
```

Comment:
Is it necessary to say this. It should be clear that Slic is part of Axom since it's in the Axom repo and the top-level Axom introduction docs (when cleaned up) will briefly describe the origins of the Axom project.